_posts/2023-03-30-semantic-science-benchmarks.md

```diff
@@ -87,224 +87,237 @@ In an earlier [blog post](https://opensearch.org/blog/semantic-search-solutions/
 
 ## Section 5: Strengths and limitations
 
-There are two ways of incorporating transformers in search: as **cross-encoders** and **neural retrievers**. 
+There are two ways of incorporating transformers in search: as **cross-encoders** and **neural retrievers**.
 
-**Cross-encoders** work in series with keyword search and can be thought of as rerankers. Once BM25 fetches the top $$N$$ results for a given query, a cross-encoder reranks these $$N$$ results given the query $$q$$. This approach generally produces better results than using neural retrievers. However, it is computationally expensive (high latency), because the transformer needs to compute $$N$$ different scores, one for each `(query, document)` pair. The cross-encoder is also limited by the result quality of the BM25 retrieval. 
+**Cross-encoders** work in series with keyword search and can be thought of as rerankers. Once BM25 fetches the top \(N\) results for a given query, a cross-encoder reranks these \(N\) results given the query \(q\). This approach generally produces better results than using neural retrievers alone. However, it is computationally expensive (high latency), because the transformer needs to compute \(N\) different scores, one for each `(query, document)` pair. The cross-encoder is also limited by the result quality of the BM25 retrieval.
 
-In contrast, **neural retrievers** have to perform only one computation: create a vector for the query. The actual retrieval is accomplished by finding the top $$N$$ nearest neighbors of this query vector. This is a very fast operation that is implemented using [k-NN](https://opensearch.org/docs/latest/search-plugins/knn/index/). Note that neural retrievers do not rely on keyword results but rather are used in combination with keyword search. Indeed, neural retrievers combined with BM25 yield better results than cross-encoders. 
+In contrast, **neural retrievers** have to perform only one computation: create a vector for the query. The actual retrieval is accomplished by finding the top \(N\) nearest neighbors of this query vector. This is a very fast operation that is implemented using [k-NN](https://opensearch.org/docs/latest/search-plugins/knn/index/). Note that neural retrievers do not rely on keyword results but rather are used in combination with keyword search. Indeed, neural retrievers combined with BM25 yield better results than cross-encoders, as shown by our experiments. It is also worth noting that cross-encoders work in the reranker paradigm and rely on first stage retrieval. They greatly benefit from combining BM25 and dense retrievers discussed in this post for their first stage retrieval. 
```

Comment:
First sentence: I feel like the colon is awkward here. Can we just say "have to perform only one computation that creates a vector for the query"? Last sentence: "They greatly benefit from combining BM25 and dense retrievers, discussed earlier in this post, for their first-stage retrieval"? Hyphenate both instances of "first-stage".